manipulation/src/utils/auto_gpu.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+// RAII structs to set CUDA device
+
+#include <string>
+#include <stdexcept>
+
+#include <ATen/ATen.h>
+
+#ifdef WITH_CUDA
+#include <cuda.h>
+#include <cuda_runtime.h>
+#endif
+
+struct AutoGPU {
```

Comment:
we probably don't need gpu access or anything related to it

Reply:
sure, I'll go through and clear more of that stuff out